### PR TITLE
(python2) Embedded package

### PR DESCRIPTION
**Pull request by jack1142**
## Description
This makes the package python2 embedded. I added legal folder with verification file and license and edited update script, so it embeds the packages.

## Motivation and Context
Package should be embedded if allowed and this app's license allows us to embed the installer (ref: #866)

## How Has this Been Tested?
I checked update.ps1, to see if it works properly, I tested the package in testing environment to see, if it installs and uninstalls properly.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [x] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by jack1142** [comment]
It works on my chocolatey test environment, so I was surprised.
From what I see though, AppVeyor's Windows image includes Python 2.7.15 already, so that might be the issue.

Ref: https://www.appveyor.com/docs/windows-images-software/#python

**Comment by AdmiringWorm** [comment]
@jack1142 I do belive you're correct with your reasoning for the failure.

File: automatic/python2/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -8,16 +8,14 @@ if ($Env:ChocolateyPackageParameters -match '/InstallDir:\s*(.+)') {
...
-  url64Bit       = 'https://www.python.org/ftp/python/2.7.15/python-2.7.15.amd64.msi'
-  checksum       = '1afa1b10cf491c788baa340066a813d5ec6232561472cfc3af1664dbc6f29f77'
-  checksum64     = '5e85f3c4c209de98480acbf2ba2e71a907fd5567a838ad4b6748c76deb286ad7'
-  checksumType   = 'sha256'
-  checksumType64 = 'sha256'
+  file           = "$toolsPath\"
+  file64         = "$toolsPath\"
 }
 
 Install-ChocolateyPackage @params
```
Comment (AdmiringWorm) [inline automatic/python2/tools/chocolateyInstall.ps1]: This line needs to be changed
```suggestion
Install-ChocolateyInstallPackage @params
```

**Comment by jack1142** [comment]
I see you assigned it to yourself, so I don't have to do anything about this PR for now, do I?

**Comment by AdmiringWorm** [comment]
@jack1142 I was actually just testing out how the suggested edit would look if I was assigned.

But yes, you don't need to do anything.

**Comment by AdmiringWorm** [comment]
@jack1142 seems like you haven't enabled 'Edit from maintainers' (or whatever it was called), when you opened the PR.

Because of this, I can't changed anything before it gets merged in, so if you wouldn't mind. Could you do the change yourself?

**Comment by jack1142** [comment]
@AdmiringWorm I always have it checked, so it must be something else :(

vivaldi_2019-01-18_19-22-13

**Comment by AdmiringWorm** [comment]
hmm, odd. Not sure what is prevent me from updating it then. Oh well, thanks for updating.

**Comment by jack1142** [comment]
@AdmiringWorm I just want to add, that with the merge of this PR, the list from #866 is ended.

**Pull request merged**